api/mainquery/views.py

```diff
@@ -0,0 +1,30 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+class MainQuery(object):
+
+    def __init__(self):
+        self.dimensions = []
```

Comment:
Doesn't it supposed to be like there is a main query and a bunch of dimensions with keywords? So I think there should be an additional string variable aside from dimensions as well.

Query = Main query + Dimensions

Reply:
you are totally right. added main query into constructor and print methods.